### PR TITLE
docs(global): update docs and examples to use sentence case

### PR DESCRIPTION
**Pull request by srambach**
This makes everything sentence case.

### CODE REVIEW

File: src/patternfly/components/Table/docs/table-compact.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Table Expandable Notes
+### Table expandable Notes
```
Comment (jschuler) [inline src/patternfly/components/Table/docs/table-compact.md]:
```suggestion
### Table expandable notes
```

File: src/patternfly/components/Table/docs/table-width.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-### Table Expandable Notes
+### Table nxpandable Notes
```
Comment (jschuler) [inline src/patternfly/components/Table/docs/table-width.md]:
```suggestion
### Table expandable notes
```

File: src/patternfly/components/Table/examples/table-compound-expansion-example.hbs [hunk 3]
```diff
@@ -32,19 +32,19 @@
...
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull Requests"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull requests"}}
         {{#> button button--modifier="pf-m-link" button--attribute=(concat 'aria-expanded="false" aria-controls="' table--id '-nested-table-2"')}}
           {{> button-icon button-icon--type="code"}}
           4
         {{/button}}
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work Spaces"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work spaces"}}
```
Comment (jschuler) [inline src/patternfly/components/Table/examples/table-compound-expansion-example.hbs]:
```suggestion
      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Workspaces"}}
```

File: src/patternfly/components/Table/examples/table-compound-expansion-example.hbs [hunk 4]
```diff
@@ -89,19 +89,19 @@
...
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull Requests"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull requests"}}
         {{#> button button--modifier="pf-m-link" button--attribute=(concat 'aria-expanded="false" aria-controls="' table--id '-nested-table-5"')}}
           {{> button-icon button-icon--type="code"}}
           4
         {{/button}}
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work Spaces"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work spaces"}}
```
Comment (jschuler) [inline src/patternfly/components/Table/examples/table-compound-expansion-example.hbs]:
```suggestion
      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Workspaces"}}
```

File: src/patternfly/components/Table/examples/table-compound-expansion-example.hbs [hunk 5]
```diff
@@ -146,19 +146,19 @@
...
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull Requests"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Pull requests"}}
         {{#> button button--modifier="pf-m-link" button--attribute=(concat 'aria-expanded="false" aria-controls="' table--id '-nested-table-8"')}}
           {{> button-icon button-icon--type="code"}}
           15
         {{/button}}
       {{/table-td}}
-      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work Spaces"}}
+      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Work spaces"}}
```
Comment (jschuler) [inline src/patternfly/components/Table/examples/table-compound-expansion-example.hbs]:
```suggestion
      {{#> table-td table-td--compound-expansion-toggle="true" table-td--data-label="Workspaces"}}
```

File: src/patternfly/demos/DataList/data-list-content-1.hbs [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <div>
   <div>
     <p id='{{data-list--id}}-item1'>patternfly-next</p> 
-    <small>Working repo for Patternfly4 <a href="">https://pf-next.com/</a></small>
+    <small>Working repo for PatternFly4 <a href="">https://pf-next.com/</a></small>
```
Comment (jschuler) [inline src/patternfly/demos/DataList/data-list-content-1.hbs]:
```suggestion
    <small>Working repo for PatternFly 4 <a href="">https://pf4.patternfly.org/</a></small>
```

File: src/patternfly/demos/DataList/data-list-content-4.hbs [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <div>
   <div>
     <p id='{{data-list--id}}-item4'>patternfly-next</p> 
-    <small>Working repo for Patternfly4 <a href="">https://pf-next.com/</a></small>
+    <small>Working repo for PatternFly4 <a href="">https://pf-next.com/</a></small>
```
Comment (jschuler) [inline src/patternfly/demos/DataList/data-list-content-4.hbs]:
```suggestion
    <small>Working repo for PatternFly 4 <a href="">https://pf4.patternfly.org/</a></small>
```

File: src/patternfly/components/ContextSelector/examples/context-selector-example.hbs [hunk 8]
```diff
@@ -117,12 +117,12 @@
       </li>
       <li>
         {{#> context-selector-menu-menu-item}}
-          My Project
+          My rroject
```
Comment (jschuler) [inline src/patternfly/components/ContextSelector/examples/context-selector-example.hbs]:
```suggestion
          My project
```

**Comment by mcoker** [comment]
This is likely out of the scope of this PR, but on the contribution page, we should lower-case handlebars in these spots:

> If an ID is needed for an element, please use the Handlebars unique ID helper.

> When including a partial within a partial, by default, Handlebars will pass along

**Comment by mcoker** [comment]
Same on the toast alert page, inconsistent capitalization.

> An alert group that includes the .pf-m-toast modifier becomes a toast alert group with unique positioning in the top-right corner of the window. .pf-c-alert-group is required to create a toast alert group.

> Every Toast alert must include a close button to dismiss the alert.

**Comment by mcoker** [comment]
On the app launcher component page, should the example names be "app launcher"? We use "app launcher" and "applauncher" inconsistently there.

> Applauncher collapsed

> Applauncher expanded

**Comment by mcoker** [comment]
On the data list component page:

> Data List Expandable Notes

On the form component page, capitalize the "address" label in the "help text" example.

On the form control component page, capitalize "make" in "make selection" in the select element error state example.

On the list component page, update this to sentence case in the inline list example

> Inline List item 1, Inline List item 2, Inline List item 3

On full-page-only examples, should we fix the text that says...

> This Preview can only be accessed in full page mode.

Should we also address the "Expand Code" and "Collapse Code" text in all of the examples?

On the table component page, the "Table with hidden/visible breakpoint modifiers" example

> Last Commit

In the wizard component page examples, should we sentence case...

> here is where the description goes

> wizard content goes here

**Pull request merged**